src/consts.js

```diff
@@ -364,7 +364,7 @@ export const STATES_WITH_ABBREVIATIONS = [
   { label: "Oklahoma", value: "OK" },
   { label: "Oregan", value: "OR" },
```

Comment:
Looks like we have another one haha -> `Oregon`

Reply:
Nice catch hah